satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -172,23 +172,13 @@ read limitoffset (
 //--- repairqueue ---//
 
 model injuredsegment (
-	key id
-
-	field id   serial64
-	field info blob
-)
+	key path
 
-create injuredsegment ( )
-
-read first (
-	select injuredsegment
+	field path text
```

Comment:
`path` is not `text`, it's a binary blob... `text` usually implies utf-8. however, it probably won't cause problems in the database

Reply:
yes, but it looks more precise, hence should we swap it?